api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/ClaimAuditor.java

```diff
@@ -27,10 +30,18 @@ public void auditNewClaim(Claim claim) {
             return;
         }
         NewClaimEvent newClaimEvent = NewClaimEvent.builder()
-                .claimantId(claim.getId())
+                .claimId(claim.getId())
                 .claimStatus(claim.getClaimStatus())
                 .eligibilityStatus(claim.getEligibilityStatus())
                 .build();
         eventLogger.logEvent(newClaimEvent);
     }
+
+    public void auditNewCard(UUID claimId, CardResponse cardResponse) {
```

Comment:
@mikeSmith08 what do you think about removing the ClaimAuditor and just using the EventLogger directly? It doesn't seem to do very much anymore.

Reply:
I think it's still worthwhile as it nicely encapsulates the creating of the Event which the calling client then doesn't need to concern itself with.